### DISCUSSION TITLE
Is it possible to add support for FileZilla

### DISCUSSION THREAD

**Discussion by ddddavid-he** [Homebrew/discussions#2727] [Tap maintenance and Homebrew development] [answered]
FileZilla is a useful ftp tool on macOS. Looking forward to Brew support for FileZilla.

**Comment by SMillerDev** [comment, marked as answer]
Yes it is, please make a pull request for it and we'll happily review it.

**Reply by neutric** [reply]
There is https://github.com/Homebrew/homebrew-cask/pull/55583 to consider, though.

**Reply by ddddavid-he** [reply]
> There is [Homebrew/homebrew-cask#55583](https://github.com/Homebrew/homebrew-cask/pull/55583) to consider, though.

This issue solved my problem
